scripts/weather.py

```diff
@@ -0,0 +1,62 @@
+import requests, json
+from datetime import datetime, timedelta
+from deepmerge import always_merger
+from scripts.config import *
+from google.cloud import storage
+
+
+def get_weather_data(query, limit=30):
```

Comment:
At runtime we shouldn't reach out to the weather API (and especially not make several calls - we will probably have our API key revoked if our traffic spikes). The ETL job already downloads this offline and puts it in a bucket, so we should access from there (and then probably cache in memory similar to `main._list_names()`).